source/manual/rename-a-country.html.md

```diff
@@ -127,24 +127,20 @@ Failing this, there is a [rake task](https://github.com/alphagov/search-api/blob
 
 ### 7. Update Specialist Publisher
 
-1. Create a content schemas [pull request](https://github.com/alphagov/govuk-content-schemas/pull/1014) to support both countries temporarily (around the data migration).
+1. Create and deploy pull requests for GOV.UK Content Schemas ([example](https://github.com/alphagov/govuk-content-schemas/pull/1014)) and Specialist Publisher ([example](https://github.com/alphagov/specialist-publisher/pull/1722/commits/79c10d173f8294fef25b07678a7e74213e78e424)) to support both countries temporarily (around the data migration).
```

Comment:
Sorry I know you didn't add this originally, but "around the data migration" seems strange phrasing. Would "during the data migration" be more appropriate?

Reply:
Sure, updated.